examples/connext_dds/xml_application_env/c++11/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+# (c) 2020 Copyright, Real-Time Innovations, Inc. All rights reserved.
+# No duplications, whole or partial, manual or electronic, may be made
+# without express written permission.  Any such copies, or revisions thereof,
+# must display this notice unaltered.
+# This code contains trade secrets of Real-Time Innovations, Inc.
+cmake_minimum_required(VERSION 3.7)
+project(rtiexamples-xml-application)
+set(CMAKE_MODULE_PATH
+    ${CMAKE_MODULE_PATH}
+    "${CMAKE_CURRENT_SOURCE_DIR}/../../../../resources/cmake"
+)
+
+# Include ConnextDdsAddExample.cmake from resources/cmake
+include(ConnextDdsAddExample)
+
+connextdds_add_application(
+    TARGET xml_application_env
+    LANG "C++11"
+    SOURCES xml_application_env.cpp
```

Comment:
Could you change replace this line by:
```
    SOURCES "${CMAKE_CURRENT_SOURCE_DIR}/xml_application_env.cpp"
```
This will help to compile this example as part of other CMakeLists.txt file

Reply:
Done.